File: compteurAppFrontend/src/models/adresseDTO.ts
export class AdresseDTO {
  rue?: string;
  numero?: string;
  codePostal?: string;
  ville?: string;
  pays?: string;
  id?: number;
  idClient?: string;

  constructor(
    ville: string,
    pays: string,
    codePostal: string,
    rue: string,
    numero: string,
    id: number,
    idClient: string
  ) {
    this.rue = rue;
    this.numero = numero;
    this.codePostal = codePostal;
    this.ville = ville;
    this.pays = pays;
    this.id = id;
    this.idClient = idClient;
  }
}

File: backendCompteurApp/src/main/java/com/compteurapp/backendcompteurapp/DTO/AdresseDTO.java
package com.compteurapp.backendcompteurapp.DTO;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class AdresseDTO {
    private Long id;
    private String rue;
    private String numero;
    private String codePostal;
    private String ville;
    private String pays;
    private String idClient;

    public AdresseDTO(String rue, String numero, String codePostal, String ville, String pays,String idClient) {
        this.rue = rue;
        this.numero = numero;
        this.codePostal = codePostal;
        this.ville = ville;
        this.pays = pays;
        this.idClient = idClient;
    }
}


